### PR TITLE
DLS-10610 - address compilation warnings

### PR DESCRIPTION
**Pull request by TomRafferty**

### CODE REVIEW

**Comment by TomRafferty** [comment]
rebuild

File: test/unit/uk/gov/hmrc/individualsemploymentsapi/service/v1/EmploymentsServiceSpec.scala [hunk 1]
```diff
@@ -63,6 +64,7 @@ class EmploymentsServiceSpec extends SpecBase with Intervals with MockitoSugar w
 
   implicit val hc: HeaderCarrier = new HeaderCarrier
 
+  @nowarn
```
Comment (LAJW) [inline test/unit/uk/gov/hmrc/individualsemploymentsapi/service/v1/EmploymentsServiceSpec.scala:67]: What warning is suppressed here and why?

**Review by LAJW — COMMENTED** [review]
I'd like to document the `@nowarn` and/or make it warning-specific.

**Comment by TomRafferty** [comment]
nice 😎

**Review by ruthdas03 — APPROVED** [review]

**Pull request merged**